src/agenda/reservation-list/index.js

```diff
@@ -190,6 +190,8 @@ class ReactComp extends Component {
       return (
         <ActivityIndicator style={{marginTop: 80}} color={this.props.theme && this.props.theme.indicatorColor} />
       );
+    } else if (this.props.reservations[this.props.selectedDay.toString('yyyy-MM-dd')] && this.props.reservations[this.props.selectedDay.toString('yyyy-MM-dd')].length === 0) {
```

Comment:
Please explain. How is this has to do with the closeCalendar() functionality?

Reply:
whoops, this line seems not related to closeCalendar method. Actually I'm adding this for my case, to add more validation for date items that have empty data. So if selected date is empty, it render empty screen instead of render list with empty items.

Reply:
Then remove it please